### ISSUE TITLE
Automatic exploration expiration is not handled well

### ISSUE THREAD

**Issue by jantjeuh83** [LoneGazebo/Community-Patch-DLL#2718]
_1. Mod version (i.e Date - 4/23):_
24-10 with the fix for #2711

_2. Mod list (if using Vox Populi only, leave blank):_
CP only

_3. Error description:_ 
Next Turn button does not advance turn when clicked.

_4. Steps to reproduce (optional):_
Load the below save game, then hit the Next Turn button.

[Ramesses II_0602 AD-1976-January.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/551649/Ramesses.II_0602.AD-1976-January.zip)

[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/551648/Logs.zip)

**Comment by LoneGazebo** [comment]
@ilteroi, related to issue I noted here: https://github.com/LoneGazebo/Community-Patch-DLL/issues/2715#issuecomment-256159689

**Comment by TechpriestEnginseer** [comment]
I've received this errors a lot if I ever clicked on the Event Overview.

**Comment by jantjeuh83** [comment]
I can advance the turn when I order the scout beneath Wellington to sleep or skip a turn. Seems like this issue in particular is caused by scouts being set to auto-explore, but no longer having any areas to explore.

@ilteroi: can you supply that minifix for auto-explorers you mentioned in #2715? Is it compatible with the fix for #2711?

**Comment by jantjeuh83** [comment]
It may also have something to do with the Satellites tech unlocking the whole map, and therefore the scout no longer having anything to explore.

**Comment by ilteroi** [comment]
try this one:

[CvGameCore_Expansion2.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/553231/CvGameCore_Expansion2.zip)

you might have to start playing the turn before the problem occurs

**Comment by jantjeuh83** [comment]
I'll give it a try.

**Comment by jantjeuh83** [comment]
Nope, that did not fix it, there is definitely still an issue with automatic exploration. Here's an interesting save file. Load it, then hit 'Next Turn'. A new Pathfinder (there are already two roaming about with automatic exploration) will be produced, and will ask for orders. Tell it to 'Explore (Automated)' by hitting E or selecting the menu option. Note that automatic exploration is then **not** activated for this new Pathfinder, probably because the entire visitable map (embarkment is not yet possible) has already been explored. However, the UI does not ask for different orders (it should), and shows the 'Next Turn' button instead. Clicking it won't do anything, however.

Telling the new Pathfinder to do nothing (space bar) two turns in a row also causes the 'Next Turn' button to show up but not function. In that case the Pathfinder in the lower right (next to Seoul) is causing the issue, and must be manually told to skip a turn. 

To summarize, the game does not handle the automatic disabling of automatic exploration well.

[Pocatello_0009 BC-3460.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/553668/Pocatello_0009.BC-3460.zip)

**Comment by ilteroi** [comment]
how about this then [CvGameCore_Expansion2.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/554475/CvGameCore_Expansion2.zip)

**Comment by jantjeuh83** [comment]
Thanks, I'll take a look later this evening.

**Comment by jantjeuh83** [comment]
That seems to have fixed it. At least, it correctly states 'A Unit Needs Orders' rather than 'Next Turn' now.
